File: Drivers/FSCC Drivers/windows/examples/fscc/clock-frequency.c
#include <fscc.h>
#include "calculate-clock-bits.h"

int main(void)
{
    HANDLE h = 0;
    DWORD tmp;
    unsigned char clock_bits[20];

    h = CreateFile("\\\\.\\FSCC0", GENERIC_READ | GENERIC_WRITE, 0, NULL,
                   OPEN_EXISTING, 0, NULL);

    /* 18.432 MHz */
    calculate_clock_bits(18432000, 10, clock_bits);

    DeviceIoControl(h, FSCC_SET_CLOCK_BITS,
                    &clock_bits, sizeof(clock_bits),
                    NULL, 0,
                    &tmp, (LPOVERLAPPED)NULL);

    CloseHandle(h);

    return 0;
}

File: Drivers/FSCC Drivers/windows/lib/fscc/net/examples/clock-frequency.cs
using Fscc;

public class Tutorial
{
    public static int Main(string[] args)
    {
        Fscc.Port p = new Fscc.Port(0);
        
        // 18.432 MHz
        p.ClockFrequency = 18432000;

        return 0;
    }
}